test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3476,7 +3479,7 @@ def forward(self, x):
     def test_scalar_type(self):
         class ArithmeticModel(torch.nn.Module):
             def forward(self, x):
-                return x.size(0) * 2 * x
+                return x.size(0) * 2 * x, 2 - x
```

Comment:
Is this test case passing for scripting as well?

Reply:
yes it passes locally.